images/base/files/usr/local/bin/entrypoint

```diff
@@ -181,13 +181,28 @@ enable_network_magic(){
   sed -e "s/${docker_embedded_dns_ip}/${docker_host_ip}/g" /etc/resolv.conf.original >/etc/resolv.conf
 
   # fixup IPs in manifests ...
-  # TODO: ensure this supports rebooting the host w/ IPv6 clusters
-  curr_ip=$(hostname --ip-address)
-  if [ -f /kind/old-ip ]; then
-    old_ip=$(cat /kind/old-ip)
-    sed -i "s#${old_ip}#${curr_ip}#" /etc/kubernetes/manifests/*.yaml /var/lib/kubelet/kubeadm-flags.env
+  curr_ipv4="$( (getent ahostsv4 $(hostname) | head -n1 | cut -d' ' -f1) || true)"
```

Comment:
I've tried to put it in an array but is too brittle, this option has more boilerplate code bit is more resilient ... the order of the returned address may change depending on the node configuration, or someone can enable ipv6 addresses later on, who knows :shrug:

Reply:
if we depended on this for something other than host reboot I'd be pretty unhappy with how much `|| true` is going on here, but given the context this seems reasonable..

Reply:
guarding against this more in another commit